servers/v3-proxy-api/src/graph/get/toRest.ts

```diff
@@ -99,6 +105,92 @@ function searchStatusResponse(
   };
 }
 
+function AccountTransformer(
+  accountData: Partial<AccountFieldsFragment>,
+): AccountResponse | Record<never, never> {
+  // Short-circuit if data is not present
+  if (accountData?.id == null) return {};
+  const firstName = accountData.firstName ?? '';
+  const lastName = accountData.lastName ?? '';
+  return {
+    account: {
+      user_id: accountData.id,
+      username: '', // omitted - included for shape only
+      email: accountData.email,
+      birth: mysqlTimeString(
+        new Date(accountData.accountCreationDate),
+        'US/Central',
+      ),
+      first_name: firstName,
+      last_name: lastName,
+      premium_status: accountData.isPremium ? '1' : '0',
+      is_fxa: accountData.isFxa ? 'true' : 'false',
+      // TODO - Fetch all aliases POCKET-9882
```

Comment:
@kschelonka moving forward with our FxA work we disabled creation of new aliases. I think its ok to not do this.